### DISCUSSION TITLE
how can i get item_purchase event?

### DISCUSSION THREAD

**Discussion by zyi1992** [saul/demofile#208] [Questions] [answered]
When use demofile i wan't to get item info , i code like this
`    demoFile.gameEvents.on('item_purchase',e=>{
      console.log(e);
    });`
I can get player_death event but can't get any item about event, is there anything wrong?
Home for you replay！

**Comment by saul** [comment]
If you can't see the `item_purchase` event, then that means that it isn't networked to clients and so unfortunately is not available in demo files.

**Comment by zyi1992** [comment]
Sorry for i can't understand what you replay, Which clients is you means, and as you said the  item_purchase event  is not available only with demo files?

**Comment by saul** [comment, marked as answer]
item_purchase is not available in demo files
